### DISCUSSION TITLE
CI/CD pipeline on a tag generates different versions than on the main branch with this tag

### DISCUSSION THREAD

**Discussion by IT-VBFK** [GitTools/GitVersion#4290] [Q&A]
Hi

Maybe this is per design, but I thought I ask it here in case it isn't.

When I tag a commit e.g. with tag 1.1.1 then the generated version is 1.2.0-1-1-1.1 while if I run a main pipeline from the same commit with the tag I correctly get 1.1.1.

Is this a misconfiguration?

BTW: I am using v6.0.2

**Comment by asbjornu** [comment]
No, this is not by design – it is a bug. #4212 discusses how we might be able to fix this.
